src/wp-includes/class-wp-scripts.php

```diff
@@ -419,14 +461,15 @@ public function do_item( $handle, $group = false ) {
 	 *
 	 * @since 4.5.0
 	 *
-	 * @param string $handle   Name of the script to add the inline script to.
-	 *                         Must be lowercase.
-	 * @param string $data     String containing the JavaScript to be added.
-	 * @param string $position Optional. Whether to add the inline script
-	 *                         before the handle or after. Default 'after'.
+	 * @param string $handle     Name of the script to add the inline script to.
+	 *                           Must be lowercase.
+	 * @param string $data       String containing the JavaScript to be added.
+	 * @param string $position   Optional. Whether to add the inline script
+	 *                           before the handle or after. Default 'after'.
+	 * @param bool   $standalone Inline script opted to be standalone or not. Default false.
```

Comment:
All optional params should start with the one word sentence, "Optional."

See: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods

```suggestion
	 * @param bool   $standalone Optional. Inline script opted to be standalone or not. Default false.
```

Reply:
Resolved.